float64[] solution_x
float64 final_obj_value
float64 t0
float64 tend
float64 tf
float64 psi0

